Lib/graphlib.py

```diff
@@ -56,40 +57,40 @@ def _get_nodeinfo(self, node):
             self._node2info[node] = result = _NodeInfo(node)
         return result
 
-    def add(self, node, *predecessors):
-        """Add a new node and its predecessors to the graph.
+    def add(self, start_node, *end_nodes):
```

Comment:
Strictly speaking, the change is not backward compatible.
The current module allows keyword notation: `topo.add(node='a', 'b', 'c', 'd')

Reply:
Ok, I reverted the name from `start_node` back to `node`.

I have made the requested changes; please review again